lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/time/Clock.java

```diff
@@ -43,12 +43,12 @@
 
   /**
    * Schedule a TimerEvent at the given future offset.
-   *
-   * @param handler to be called
-   * @param offset  into the future
-   * @throws IllegalStateException when the clock has been already closed
+   * @param handler Event handler to be called on alarm.
+   * @param offset Offset into the future in milliseconds.
+   * @return Newly scheduled alarm.
+   * @throws IllegalStateException When the clock has been already closed.
    */
-  void scheduleAlarm(final int offset, final EventHandler<Alarm> handler);
+  Time scheduleAlarm(final int offset, final EventHandler<Alarm> handler);
```

Comment:
@markusweimer I don't think this one should - adding a return value does not change the way the method was used before, when it was void.